python/pipeline/stack.py

```diff
@@ -512,7 +511,6 @@ class ROICoordinates(dj.Part):
         -> Stitching.Volume             # volume to which this ROI belongs
         stitch_xs        : blob         # (px) center of each slice in the volume-wise coordinate system
         stitch_ys        : blob         # (px) center of each slice in the volume-wise coordinate system
-        stitch_z         : float        # (um) initial depth in the motor coordinate system
```

Comment:
Because I'm not actually changing the z when I do the stitching. I only stitch in y and x.  But two field of views that were supposedly recorded at the same depth are assumed to be at the same depth. This is ok for mesoscope scans (the main and only use for stitching till now): two overlapping field-of-views may be a bit off in x, y but they are pretty much the same in z